packages/admin-ui/src/pages/dashboard/components/HostStats.tsx

```diff
@@ -18,22 +18,24 @@ const StatsCardContainer: React.FunctionComponent<{
   return (
     <Card className="stats-card">
       <div className="header">
-        <span className="id">{title}</span> <span className="usage">usage</span>
+        <span className="id">{title}</span>{" "}
       </div>
       {children}
     </Card>
   );
 };
 
-function StatsCardOk({ percent, text }: { percent: number; text?: string }) {
-  const value = Math.round(percent);
+function StatsCardOk({ value, text, valueType }: { value: number; text?: string; valueType: "percentage" | "temperature" }) {
+  const valueBar = Math.round(value);
 
   return (
     <>
       <ProgressBar
         variant={parseVariant(value)}
-        now={value}
-        label={value + "%"}
+        now={valueBar}
+        label={ valueType==="percentage" ? valueBar + "%"
```

Comment:
why not putting as argument directly the label?
```js
{ label: string}
```
and then calling the function with either the label "%" or "ºC", and if undefined take the `valueBar`